src/shared/sam/localLambdaRunner.ts

```diff
@@ -228,7 +228,7 @@ async function invokeLambdaHandler(
             debugPort: debugPort,
             debuggerPath: config.debuggerPath,
             debugArgs: config.debugArgs,
-            skipPullImage: config.sam?.skipNewImageCheck,
+            skipPullImage: true, // We already built the image, but `sam local start-api` will try to build it again
```

Comment:
good idea. should we also do this for the `SamCliLocalInvokeInvocationArguments` case below?

Reply:
Yeah makes sense to me. Still can't figure out why the very first CodeLens fails though. Strange.